modal_proto/api.proto

```diff
@@ -2088,6 +2088,7 @@ message WebhookConfig {
   repeated CustomDomainConfig custom_domains = 6;
   uint32 web_server_port = 7;
   float web_server_startup_timeout = 8;
+  bool fastapi_endpoint_docs = 9;
```

Comment:
We would kind of like to move away from fastapi for web endpoint to reduce the client dependency burden. So I might suggest naming this more generally.

That also raises the question of whether it’s a good idea to expand the “minimum viable web endpoint” functionality, as it ties us closer to relatively heavy dependencies. That would be my one hesitation about adding this as an opt-in feature.

Reply:
I named it this way because (per Slack discussions and Linear ticket) I expect it to migrate onto a higher-level `fastapi_endpoint` decorator in the future that wraps a lower-level `web_endpoint` that wouldn't have these bells and whistles. It seems like it's slightly annoying to rename fields in protobufs, so I thought I'd jump ahead to the intended final name.

Reply:
I see, although would we necessarily only expose endpoint docs when using `fastapi`? We may roll our own lightweight functionality and decide it's feasible to include some kind of documentation in that featureset.